docs/General/getting-started/getting-started-with-bpmn-js.md

```diff
@@ -10,3 +10,254 @@ grand_parent: "General"
+
+#### Including the Pre-packaged Version
+
+For the pre-packaged version, we should include the bpmn-js modeler via HTML script tag:
+
+```html
+<!-- Feel free to replace the CDN url with local bpmn-js path if necessary -->
+<script src="https://unpkg.com/bpmn-js/dist/bpmn-modeler.production.min.js"></script>
+```
+
+#### Including via NPM
+
+First, we should install bpmn-js via NPM:
+```
+npm install bpmn-js
+```
+
+We shoyld then import the BPMN modeler as an ES module:
+
+```javascript
+import Modeler from 'bpmn-js/lib/Modeler';
+```
+
+#### Instantiating the Modeler
+
+Just like we did with the viewer, we need to create an instance of bpmn-js modeler after the setup.
+
+If we've imported the bpmn-js pre-packaged:
+```javascript
+const modeler = new BpmnJS({ container: '#canvas' });
+```
+
+if we've imported the bpmn-js via NPM, we should go for the `Modeler` constructor instead:
+```javascript
+const modeler = new Modeler({ container: '#canvas' });
+```
+
+Just like we did with the viewer, we'll import an initial XML to get the modeler going. We'll assume the same BPMN file we used above for the viewer:
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<bpmn:definitions xmlns:bpmn="http://www.omg.org/spec/BPMN/20100524/MODEL" xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI" xmlns:dc="http://www.omg.org/spec/DD/20100524/DC" id="Definitions_0xcv3nk" targetNamespace="http://bpmn.io/schema/bpmn" exporter="Camunda Modeler" exporterVersion="3.0.0">
+  <bpmn:process id="Process_0sckl64" isExecutable="true">
+    <bpmn:startEvent id="StartEvent_1" />
+  </bpmn:process>
+  <bpmndi:BPMNDiagram id="BPMNDiagram_1">
+    <bpmndi:BPMNPlane id="BPMNPlane_1" bpmnElement="Process_0sckl64">
+      <bpmndi:BPMNShape id="_BPMNShape_StartEvent_2" bpmnElement="StartEvent_1">
+        <dc:Bounds x="179" y="159" width="36" height="36" />
+      </bpmndi:BPMNShape>
+    </bpmndi:BPMNPlane>
+  </bpmndi:BPMNDiagram>
+</bpmn:definitions>
+```
+
+```javascript
+const xml = ' ... '; // We'll assume this variable contains the XML content above.
+
+try {
+
+  await modeler.importXML(xml);
+
+  console.log('Modeler is ready');
+} catch (err) {
+
+  console.log('An error happened importing XML:', err);
+}
+```
+
+#### Putting It All Together
+
+See this [Sandbox](https://codesandbox.io/s/quiet-pine-182uh?file=/src/index.js).
+
+## Dynamic Attaching/Detaching
+
+bpmn-js allows attaching the viewer/modeler dynamically via [attachTo API](/general/api-reference/bpmn-js/attachTo) to any DOM element:
+
+```javascript
+// We'll assume the bpmnJS is an instance of Modeler or a Viewer
+
+bpmnJS.attachTo('#some-other-element');
+```
+
+We can also detach it from any element via [detach API](/general/api-reference/bpmn-js/detach):
+
+```javascript
+bpmnJS.detach();
+```
+
+## Going Further
+
+There's no shortage of crazy things one can do with bpmn-js. Check these pages out in order to go further with this library:
```

Comment:
In technical documentation, I don't want to read about _crazy things on can do_.

Reply:
Done 👍